auto_submit/lib/service/config.dart

```diff
@@ -41,24 +41,42 @@ class Config {
 
   Future<GitHub> createGithubClient() async {
     // GitHub's secondary rate limits are run into very frequently when making auth tokens.
-    final String token = await cache['githubToken'].get(
+    final Uint8List? cacheValue = await cache['githubToken'].get(
       _generateGithubToken,
       // Tokens have a TTL of 10 minutes. AppEngine requests have a TTL of 1 minute.
       // To ensure no expired tokens are used, set this to 10 - 1, with an extra buffer of a duplicate request.
       const Duration(minutes: 8),
     );
-
+    final String token = String.fromCharCodes(cacheValue!);
     return GitHub(auth: Authentication.withToken(token));
   }
 
+  Future<String> _getInstallationId() async {
+    final String jwt = await _generateGithubJwt();
```

Comment:
Can we add a test?

Reply:
Sure Casey, I added a unit test for the config class.